src/Conversion/ONNXToTOSA/DialectBuilder.cpp

```diff
@@ -182,6 +182,36 @@ Value TosaBuilder::mul(mlir::Value &lhs, mlir::Value &rhs, int32_t shift) {
       rewriter(), loc(), newValueType, lhs, rhs, shift);
 }
 
+Value TosaBuilder::div(mlir::Value &lhs, mlir::Value &rhs) {
```

Comment:
There was the suggestion to rename `div` to `intdiv` or something of the sort; was the comment acted upon (saw a thumbs up, which seems to indicate that this was agreed? )

Reply:
My comment was about renaming the TOSA dialect op from DivOp to IntDivOp, but that needs to happen upstream and then updated in onnx-mlir along with the LLVM merge that includes the upstream change. We could change the method here to TosaBuilder::intdiv now to minimize the future work and be more clear at the TosaBuilder interface today.